Airship.Android.Chat.nuspec

```diff
@@ -11,14 +11,14 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid4.1">
-            <dependency id="airship.android.core" version="16.7.1"/>
+            <dependency id="airship.android.core" version="16.7.2"/>
             <dependency id="Xamarin.Kotlin.StdLib" version="1.5.31.2" />
             <dependency id="Xamarin.Kotlin.StdLib.Jdk8" version="1.5.31.2" />
             <dependency id="Xamarin.KotlinX.Coroutines.Core" version="1.5.2.2" />
             <dependency id="Xamarin.KotlinX.Coroutines.Android" version="1.5.2.2" />
             <dependency id="Xamarin.AndroidX.ConstraintLayout" version="2.1.1.2" />
             <dependency id="Xamarin.AndroidX.Paging.Runtime" version="2.1.2.7" />
-            <dependency id="Xamarin.AndroidX.Room.Runtime" version="2.3.0.4"/>
+            <dependency id="Xamarin.AndroidX.Room.Runtime" version="2.4.1"/>
```

Comment:
I think this should be `2.4.3` to match the Room version used in Chat

Reply:
Yes 👍